pyccel/ast/builtins.py

```diff
@@ -881,9 +881,10 @@ def print_string(self):
         can be used in a print  statement
         """
         prec = self.precision
+        str_dtype = str(self.dtype)
         return LiteralString("<class '{dtype}{precision}'>".format(
-            dtype = str(self.dtype),
-            precision = '' if prec in (None, -1) else prec*8))
+            dtype = str_dtype,
+            precision = '' if prec in (None, -1) else (prec*8*[1, 2][str_dtype=="complex"])))
```

Comment:
This like could be changed to something like this i think:
```python
precision = '' if prec in (None, -1) else (prec * 8 * (2 if str_dtype=="complex" else 1))))
```

Reply:
fixed here: https://github.com/pyccel/pyccel/pull/1167/commits/8da5291f3e1bf05cf26c039b8f53ad1b16cd5a9d